cmd/manifest/init-v1.go

```diff
@@ -141,7 +141,7 @@ func (*ManifestCommand) RunInitV1(ctx context.Context, opts *InitOpts) error {
 		}
 
 		if !supportsPersistentVolumes(ctx) {
-			oktetoLog.Yellow("Default storage class not found in your cluster. Persistent volumes not enabled in your okteto manifest")
+			oktetoLog.Yellow("Default storage class not found in your context. Persistent volumes not enabled in your okteto manifest")
```

Comment:
```suggestion
			oktetoLog.Yellow("Default storage class not found in your cluster. Persistent volumes not enabled in your okteto manifest")
```
I thing in this case is clearer talk about cluster